packages/moonstone/Picker/Picker.js

```diff
@@ -187,14 +187,17 @@ const PickerBase = kind({
 				validateRange(value, 0, max, 'Picker', '"value"', 'min', 'max index');
 			}
 			return clamp(0, max, value);
+		},
+		voiceLabel: ({children}) => {
+			return JSON.stringify(children.map((child) => (typeof child === 'object' ? child.props.children : child)));
```

Comment:
I don't think this is a safe way to extract the values from `children`.  Any arbitrary node could be passed and calling `stringify` on an unknown object is not a very good idea.  Perhaps only handle the most common cases and don't supply a voice label for others?

Reply:
- As you mentioned, error can be occur when unexpected children is used in Picker(Thanks for comment about it)
- I did check that how to picker is used in application side. In most case, they declare array at children. But there is still possibility that div(object) can be used.
- Basically, I try to set data-webos-voice-labels-ext automatically refer to children if the item's type is string or number.
- Exceptional case, we can't gain proper children label value only component side. In this case, app developer has to declare 'data-webos-voice-labels-ext' attribute and it will be work as expected.